data/transactions/logic/eval_test.go

```diff
@@ -911,7 +911,11 @@ const globalV4TestProgram = globalV3TestProgram + `
 `
 
 const globalV5TestProgram = globalV4TestProgram + `
-// No new globals in v5
+global GroupID
+int 32
+bzero
+==
+&&
```

Comment:
Ideally we should also have a test where GroupID is not empty...

Reply:
That should be doable with some hacking around in evalStateful_test.go, which builds up some groups with two txns in them.